bedrock/settings/base.py

```diff
@@ -1039,6 +1040,22 @@ def get_app_name(hostname):
 CONTENTFUL_NOTIFICATION_QUEUE_SECRET_ACCESS_KEY = config("CONTENTFUL_NOTIFICATION_QUEUE_SECRET_ACCESS_KEY", default="", raise_error=False)
 CONTENTFUL_NOTIFICATION_QUEUE_WAIT_TIME = config("CONTENTFUL_NOTIFICATION_QUEUE_WAIT_TIME", default="10", parser=int, raise_error=False)
 
+CONTENTFUL_HOMEPAGE_LOOKUP = {
```

Comment:
Perhaps we should use these in the home page view as well, just to make sure this remains accurate.

Reply:
We do now use them in `bedrock.mozorg.views.home_view`, if that's what you mean:

eg https://github.com/mozilla/bedrock/pull/11061/files#diff-6cfdf8d97afa7adfe1b9d1ff09b2c25192abf7a08f60e6bd8d3efe6497625ad5R135

```
ctx.update(ContentfulEntry.objects.get_page_by_id(content_id=settings.CONTENTFUL_HOMEPAGE_LOOKUP["en-US"]))
```

Reply:
Doh! Meant to delete this comment before posting the review. I did find that. Thanks.